CHANGELOG.md

```diff
@@ -4,6 +4,63 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc1 (2020-10-06)
+
+This release adds major new features and bugfixes since the 1.7.4 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for distributed hypertables to
+TimescaleDB. With 2.0, users can run multiple instances of TimescaleDB,
+configured so that one serves as an access node and multiple others as data
+nodes, so that all queries for a distributed hypertable are issued to the access
+node, but inserted data and queries are pushed down across data nodes for
+greater scale and performance.
```

Comment:
eliminate so that, so that repetition:

```suggestion
TimescaleDB. With 2.0, users can create distributed hypertables across 
multiple instances of TimescaleDB, 
configured so that one instance serves as an access node and multiple others as data
nodes. All queries for a distributed hypertable are issued to the access
node, but inserted data and queries are pushed down across data nodes for
greater scale and performance.
```